contracts/token/account_impl.cpp

```diff
@@ -135,6 +139,9 @@ void account_impl::sub_deposit(extended_asset value) {
 
 void account_impl::add_deposit(extended_asset value) {
    if (!exists()) {
+      check(!_st->option(token_impl::opt::floatable) ||
```

Comment:
This check should be performed before `line 141: if (!exists()) {`. In case `add_deposit` to existing account, check is bypassed.